.github/workflows/test.yml

```diff
@@ -25,6 +81,32 @@ jobs:
         uses: actions/checkout@v3
       - name: Run tests with Thread Sanitizer
         run: swift test --sanitize=thread
-        env:
-          MONGO_HOSTNAME_A: mongo-a
-          MONGO_HOSTNAME_B: mongo-b
+
+  macos-all:
+    if: github.event_name == 'pull_request'
+    strategy:
+      fail-fast: false
+      matrix:
+        dbimage: ['mongodb-community']
+        macos: ['macos-11', 'macos-12']
+        xcode: ['latest-stable']
+    runs-on: ${{ matrix.macos }}
+    env:
+      MONGO_HOSTNAME: 127.0.0.1
+      MONGO_HOSTNAME_A: 127.0.0.1
```

Comment:
Are the ports at least different? I'm not sure what impact/benefit this has.

Reply:
In Mongo's case, the default values for the `MONGO_DATABASE_A` and `MONGO_DATABASE_B` environment variables specify two different database names (see [FluentMongoDriverTests.swift:253](https://github.com/vapor/fluent-mongo-driver/blob/main/Tests/FluentMongoDriverTests/FluentMongoDriverTests.swift#L253) and [FluentMongoDriverTests.swift:261](https://github.com/vapor/fluent-mongo-driver/blob/main/Tests/FluentMongoDriverTests/FluentMongoDriverTests.swift#L261)). The end result is that whether the A and B hostnames are the same or not, the two connections operate in different namespaces. This is sufficient for the needs for the relevant tests.